config/locales/es.yml

```diff
@@ -336,6 +353,14 @@ es:
       remove_meeting_subject: 'if me | %{user} ha cancelado "%{meeting}" de "%{group}"'
       join_meeting_subject: 'if me | %{user} se ha unido a "%{meeting}" de "%{group}"'
       join_meeting_body: '<p>Para ver "%{meeting}", %{link}!</p>'
+      reported_user_subject: 'if me | Usted ha informado de un usuario'
+      reported_user_body: 'Usted ha informado %{reported_name}. Esto será revisado por un administrador.'
+      reportee_user_subject: 'if me | Has sido reportado'
+      reportee_user_body: 'Has sido reportado. Esto será revisado por un administrador.'
+      add_ban_subject: 'if me | Has sido prohibido'
+      add_ban_body: 'Has sido prohibido por violar a nuestra %{code_of_conduct_link}.'
+      remove_ban_subject: 'if me | Ya no estas prohibido'
+      remove_ban_body: 'Usted fue previamente prohibido por violar a nuestra %{code_of_conduct_link}. Ya no estás prohibido.'
```

Comment:
I would rephrase this to something like "Anteriormente fuiste baneado/a por violar nuestro %{code_of_conduct_link}. Ya no estás baneado/a" or "El ban aplicado sobre tu cuenta por violar nuestro %{code_of_conduct_link} ha sido cancelado".